archetypes/archetypes/src/main/archetype/mp/custom/database-outputs.xml

```diff
@@ -153,6 +153,10 @@
                             <value key="artifactId">helidon-integrations-cdi-datasource-hikaricp</value>
                             <value key="scope">runtime</value>
                         </map>
+                        <map>
+                            <value key="groupId">org.slf4j</value>
+                            <value key="artifactId">slf4j-simple</value>
```

Comment:
We should probably use `slf4j-jdk14`.
When we work #7738 the dependency will be a choice.

Reply:
Done